src/components/Spacer/Spacer.tsx

```diff
@@ -2,22 +2,24 @@ import React, { FunctionComponent, ReactNode } from 'react';
 
 import classNames from 'classnames';
 
+type SpacerMarginOption =
+	| 'small'
```

Comment:
leading pipe can be removed

Reply:
Pretty sure dat tslint deze altijd generate.

Reply:
ok, geen probleem dan